bin/sozo/src/commands/print_env.rs

```diff
@@ -0,0 +1,58 @@
+use anyhow::Result;
+use clap::Args;
+use dojo_world::metadata::dojo_metadata_from_workspace;
+use scarb::core::Config;
+use tracing::trace;
+
+use super::options::account::AccountOptions;
+use super::options::starknet::StarknetOptions;
+use super::options::world::WorldOptions;
+
+#[derive(Debug, Args)]
+pub struct PrintEnvArgs {
+    #[command(flatten)]
+    account: AccountOptions,
+
+    #[command(flatten)]
+    starknet: StarknetOptions,
+
+    #[command(flatten)]
+    world: WorldOptions,
+}
+
+impl PrintEnvArgs {
+    pub fn run(self, config: &Config) -> Result<()> {
+        trace!("Executing PrintEnv command.");
```

Comment:
Let's try to namespace all our loggers

Reply:
i think a better way would be to use [`span's`](https://docs.rs/tracing/latest/tracing/#spans) for this usecase, instead of printing in each of them. lmk if you have more thoughts on this for now i will add spans.